src/App.php

```diff
@@ -282,7 +291,7 @@ public function url($args = [])
         $page = $args[0];
         unset($args[0]);
 
-        $url = $page ? ($page.'.php') : '';
+        $url = $page ? ($page.($extension ? '.'.$extension : '')) : '';
```

Comment:
I think we should rename url() into page() in the future. The full name could be getPageURL().